src/validation/FormValidator.ts

```diff
@@ -0,0 +1,29 @@
+import Validator from 'validatorjs';
+import ruleTranslations from './validationRuleTranslations';
+
+export interface IValidationResult {
+    isValid: boolean;
```

Comment:
Yeah, you are correct. The reason why I did it this way, is since validatorjs official function for validating is .passes(). and therefore I would like to use this function. By only using error message, we are not using their preferred way to validate, and I'm not certain about the results in that case. I don't for example know, if the validator can invalidate a field, but not be able to give an error message.

Reply:
So, I would like to keep it this way